packages/devtools-reps/src/object-inspector/components/ObjectInspector.css

```diff
@@ -28,17 +28,23 @@
 }
 
 .tree.object-inspector .block .object-label:before {
-  content: "☲ ";
-  font-size: 1.1em;
+  content: "☲";
+  font-size: 1em;
+  display: inline-block;
+  margin-top: -1px;
+  vertical-align: text-top;
+  padding-inline-start: 3px;
```

Comment:
It looks like you're pushing this character to the right by 3px. How does it work if the rest of the text wraps to a second line? Does it look like a strange first-line indent? If so, maybe those 3px should belong to the parent element or some container?

Reply:
This was meant to be `padding-inline-end`. Fixed this in a recent commit.